net-im/psi/metadata.xml

```diff
@@ -16,7 +16,7 @@
 		<flag name='extras'>Enable Psi+ patcheset</flag>
 		<flag name='hunspell'>Enable spellchecking using <pkg>app-text/hunspell</pkg></flag>
 		<flag name='iconsets'>Install additional iconsets, some of them has not clear licensing</flag>
-		<flag name='sql'>Enable psi-new-history extras patch</flag>
+		<flag name='keyring'>Enable storing password in system keyring manager</flag>
 		<flag name='webengine'>Use <pkg>dev-qt/qtwebengine</pkg> instead of <pkg>dev-qt/qtwebkit</pkg></flag>
 		<flag name='webkit'>Enable themed, html-based chatlogs</flag>
```

Comment:
Please use double quotes everywhere in this file.

Reply:
fixed